Automattic/jetpack
# VideoPress: improve blocks building process

Fixes https://github.com/Automattic/jetpack/issues/28016

#### Changes proposed in this Pull Request:

* Add `block.json` files to the `build`directory, for every block registered on the plugin
* Add `extensions/index.json` file to the `build` directory
* Update code references to the JSON files so they point to the `build` version
* Remove `src/client` code from the final plugin bundle
* Integrate extended block scripts

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Launch a new JN site with VideoPress standalone plugin and connect it (you can use the free plan)
* Go to the editor
* Look for the VideoPress block:

<img width="401" alt="Screen Shot 2022-12-20 at 16 31 15" src="https://user-images.githubusercontent.com/6760046/208751286-8b78d403-d247-4d17-9463-0f5c043cf01c.png">

* Confirm that you can add the block to the editor
* Confirm that you can upload one video and use it
* Confirm that you can edit the video informations on the right side sidebar:

<img width="700" alt="Screen Shot 2022-12-20 at 16 34 53" src="https://user-images.githubusercontent.com/6760046/208751542-69f19dc4-d9bb-4ca4-ad07-53fb3239cca1.png">

* In other words, confirm that block is present and working as expected
* Do the same testing on WPCOM
